src/security-server/admin-service/ui/src/plugins/i18n.ts

```diff
@@ -25,8 +25,7 @@
  */
 import { prepareI18n } from '@niis/shared-ui';
 
-export const availableLanguages = ['en', 'es', 'et', 'ru', 'tk'];
-
+export const availableLanguages = ['en', 'es', 'et', 'ru', 'tk','pt']; // adicionado suporte para pt (Português do Brasil)
```

Comment:
Please use `pt-BR`, we are adding support for regions too

```suggestion
export const availableLanguages = ['en', 'es', 'et', 'ru', 'tk','pt-BR'];
```